lib/internal/webstorage.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const {
+  MathAbs,
+  ObjectDefineProperty,
+  StringPrototypeCharCodeAt,
+} = primordials;
+const { emitExperimentalWarning } = require('internal/util');
+const { kConstructorKey, Storage } = internalBinding('webstorage');
+const { basename, join } = require('path');
+
+emitExperimentalWarning('Web storage');
+
+module.exports = { Storage };
+
+let lazyLocalStorage;
+let lazySessionStorage;
+
+ObjectDefineProperty(module.exports, 'localStorage', {
```

Comment:
non-blocking nit: could be simplified slightly by combining these into a single `ObjectDefineProperties` call that sets both.